pkg/cmd/infra/router/router.go

```diff
@@ -37,35 +54,75 @@ type templateRouterConfig struct {
 	StatsUsername      string
 }
 
+// f5RouterConfig is the config necessary to start an F5 router plugin
+type f5RouterConfig struct {
```

Comment:
Ok, if the intention is to support it longer term let's move the field up to the generic config so we're prepped for it and create a follow up issue to track it.